lib/gui/pages/finish/templates/success.tpl.html

```diff
@@ -15,20 +15,20 @@ <h3 class="title"><span class="tick tick--success space-right-tiny"></span> Flas
     <div class="box center">
       <div class="fallback-banner">
         <div class="caption caption-big">Thanks for using
-          <svg-icon path="../../../assets/etcher.svg"
-            width="150px"
-            height="auto"
+          <svg-icon path="'../assets/etcher.svg'"
+            width="'150px'"
+            height="'auto'"
             os-open-external="https://etcher.io?ref=etcher_offline_banner"></svg-icon>
```

Comment:
I (wrongly) assumed it didn't work and did some preliminary work on getting them out of the directive, but it turns out only 'native' properties are removed; `disabled` specifically here. But what @lurch said is something I had in mind when doing it as well so I should complete it.